surround-cli/setup.py

```diff
@@ -0,0 +1,40 @@
+"""
+Python package definition for Surround CLI
+"""
+from setuptools import setup
+
+# Collect version from VERSION file
+with open('VERSION') as version_file:
+    VERSION = version_file.read().strip()
+
+with open("requirements.txt") as f:
+    INSTALL_REQUIRES = f.read().split("\n")
+
+setup(name='surround-cli',
+      version=VERSION,
+      description='Command line interface for Surround',
+      url='https://github.com/a2i2/surround',
+      author='Scott Barnett',
+      author_email='scott.barnett@deakin.edu.au',
+      include_package_data=True,
+      packages=[
+          'surround_cli',
+          'surround_cli.checkers',
+          'surround_cli.remote',
+          'surround_cli.split',
+          'surround_cli.visualise',
+          'surround_cli.data',
+          'surround_cli.data.cli',
+          'surround_cli.configuration',
+          'surround_cli.experiment',
+          'surround_cli.experiment.web',
+          'templates'
+      ],
+      entry_points={
+          'console_scripts': [
```

Comment:
When we `pip3 uninstall surround-cli` can we also remove the generated console script?